packages/devtools_app/lib/src/framework/scaffold.dart

```diff
@@ -452,7 +452,7 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     final painter = TextPainter(
       text: TextSpan(
         text: title,
-        style: textTheme.headline6,
+        style: textTheme.titleLarge,
```

Comment:
Should this be `headlineSmall`?

Reply:
Nope, titleLarge.
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/text_theme.dart#L330